files/ko/web/css/_colon_blank/index.md

```diff
@@ -7,8 +7,7 @@ l10n:
 
 {{CSSRef}}{{SeeCompatTable}}
 
-> [!NOTE] 
-> `:blank` 선택자는 CSSWG가 지속적으로 변경하고 있음에 따라 위험 요소로 간주됩니다.
+> [!NOTE] > `:blank` 선택자는 CSSWG가 지속적으로 변경하고 있음에 따라 위험 요소로 간주됩니다.
```

Comment:
For note cards whose paragraph starts with `[`, `{`, `` ` ``, `_` or `*`, we should temporarily use the legacy syntax instead of the GFM syntax, see: https://github.com/mdn/content/pull/33384#discussion_r1588548337

```suggestion
> **참고:** `:blank` 선택자는 CSSWG가 지속적으로 변경하고 있음에 따라 위험 요소로 간주됩니다.
```